js/mapfilter/print_pane/print_pane.js

```diff
@@ -25,6 +25,7 @@ module.exports = require('backbone').View.extend({
 
     this.mapPane = new MapPane({
       el: this.$('#map-print'),
+      config: options.config,
```

Comment:
Special case because this object is created outside of `appview.js`, so when creating MapPane, need to pass along the config.  Again, could just use window.config, but prefer explicit.